.env

```diff
@@ -0,0 +1,101 @@
+### Documentation available at https://wodby.com/docs/stacks/wordpress/local
+### Changelog can be found at https://github.com/wodby/docker4wordpress/releases
+### Images tags format explained at https://github.com/wodby/docker4wordpress#images-tags
+
+### PROJECT SETTINGS
+
+DOCROOT=/var/www/html
+BASE_DIR=/opt/projects/mollie/WooCommerce
+PROJECT_NAME=my_wordpress_project
+PROJECT_BASE_URL=wp.docker.localhost
+PROJECT_MOUNT_PATH=/var/www/html/wp-content/plugins/WooCommerce
+
+DB_NAME=wordpress
+DB_USER=wordpress
+DB_PASSWORD=wordpress
+DB_ROOT_PASSWORD=password
+DB_HOST=mariadb
+
+### --- MARIADB ----
+
+MARIADB_TAG=10.3-3.4.11
+#MARIADB_TAG=10.2-3.4.11
+#MARIADB_TAG=10.1-3.4.11
+
+### --- VANILLA WORDPRESS ----
+
+WORDPRESS_TAG=5-4.16.1
```

Comment:
I would use 3.8 version until we decide what to do.

Reply:
Given that right now we can only test against one WP version, I believe that latest stable is more beneficial. More people have WP 5.2 than any other version, according to [official statistics][1]. Once we can test against multiple versions (not before we have the integration with Codeship), I am all for including 3.8.

[1]: https://wordpress.org/about/stats/#wp_versions